src/capabilities/capabilities.ts

```diff
@@ -31,13 +31,17 @@ export enum Capability {
   CODE_REVIEW = "vscode.code-review",
   SAVE_SNIPPETS = "save_snippets",
   BETA_CAPABILITY = "beta",
+  ATTRIBUTION = "attribution",
 }
 
 let enabledCapabilities: Record<string, boolean> = {};
 
 export function isCapabilityEnabled(capability: Capability): boolean {
   return enabledCapabilities[capability];
 }
+export function isAnyCapabilityEnabled(...capabilities: Capability[]): boolean {
+  return capabilities.some((capability) => !!enabledCapabilities[capability]);
```

Comment:
Why not use `includes`?

Reply:
because I want at least one of the input capabilities to be in `enabledCapabilities`